Index: core/modules/system/system.install
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- core/modules/system/system.install	(revision 3618683e8f4d56095d506078df1a9edebaea9f37)
+++ core/modules/system/system.install	(revision )
@@ -55,7 +55,7 @@
     $enabled_modules = \Drupal::moduleHandler()->getModuleList();
     foreach ($enabled_modules as $module => $data) {
       $info = system_get_info('module', $module);
-      if ($info['package'] === 'Core (Experimental)') {
+      if (!empty($info['package']) && $info['package'] === 'Core (Experimental)') {
         $experimental[$module] = $info['name'];
       }
     }
