Index: core/modules/text/text.module
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- core/modules/text/text.module	(date 1477316688000)
+++ core/modules/text/text.module	(revision )
@@ -81,11 +81,13 @@
 
   // Retrieve the filters of the specified text format, if any.
   if (isset($format)) {
-    $filters = FilterFormat::load($format)->filters();
     // If the specified format does not exist, return nothing. $text is already
     // filtered text, but the remainder of this function will not be able to
     // ensure a sane and secure summary.
-    if (!$filters) {
+    if (!($filter_format = FilterFormat::load($format))) {
+      return '';
+    }
+    if (!($filters = $filter_format->filters())) {
       return '';
     }
   }
